ionic2/app/pages/about/about.page.ts

```diff
@@ -1,12 +1,14 @@
 import { Page, NavController } from 'ionic-angular';
+import { PEOPLE } from './about-people';
 
 @Page({
   templateUrl: 'pages/about/about.page.html',
 })
 export class AboutPage {
+  people: Object
 
   constructor(private navCtrl: NavController) {
```

Comment:
I think we can get rid of the `NavController` parameter as long as we don't need it